recipe/meta.yaml

```diff
@@ -53,6 +55,19 @@ outputs:
       commands:
         - pypy --version
 
+  - name: cffi
+    version: {{ cffi_version }}
+    build:
+      number: {{ build_num }}
```

Comment:
I sometimes have the feeling that anaconda.org only supports one package per `subdir` and build-string (without the hash). This probably won't be a problem here as `cffi` is not `noarch` bit it would still be good to have `pypy` in the build string.

Reply:
Done. Thanks